internal/ui/backup/progress.go

```diff
@@ -153,9 +155,13 @@ func (p *Progress) Run(ctx context.Context) error {
 			}
 
 			if p.totalCh == nil {
-				secs := float64(time.Since(p.start) / time.Second)
-				todo := float64(total.Bytes - processed.Bytes)
-				secondsRemaining = uint64(secs / float64(processed.Bytes) * todo)
+				rate := estimator.rate(time.Now())
+				if rate <= 0 {
```

Comment:
I don't think the ETA is useful any more once the rate falls below a few KB/s. So maybe use `rate < 1024` here?